src/views/v2/list/nav/next.php

```diff
@@ -21,14 +21,12 @@
 		rel="next"
 		class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium"
 		data-js="tribe-events-view-link"
-	>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__next-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
+	><?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: de1cc5a9a6871689a93cbe3e1f680894</sub>